umbra-js/src/utils/utils.ts

```diff
@@ -316,12 +306,11 @@ async function resolveEns(name: string, provider: EthersProvider) {
 /**
  * @notice Attempt to resolve a CNS name, and return null on failure
  * @param name
- * @param provider
  * @returns
  */
-async function resolveCns(name: string, provider: EthersProvider) {
+async function resolveCns(name: string) {
   try {
-    const resolution = getResolutionInstance(provider);
+    const resolution = getResolutionInstance();
     const address = await resolution.addr(name, 'ETH');
```

Comment:
Does this return a checksum address? We try to use with checksummed addresses everywhere

Reply:
I don't think so, I am now converting this address into checksum address using getAddress function of ethersjs

![image](https://user-images.githubusercontent.com/72156537/191677872-d4a8ff92-ce59-4c1c-89b5-eb0cf402fc12.png)

Reply:
Nice that looks perfect! (Though I don't think you pushed that change)